ntbs-service/wwwroot/css/site.scss

```
@@ -178,6 +178,14 @@ body {
     @media (min-width: 40.0625em) {
         margin-top: 18px;
         margin-bottom: 40px;
-        font-size: 16px;
+
+        /* 
+            html structure is `div.nhsuk-table responsive > table.nhsuk-table.reduced-table`
```

```suggestion
            html structure is `div.nhsuk-table-responsive > table.nhsuk-table.reduced-table`
```
Is that what we're looking for?

though in general, if that's case, should we just target it directly with a top level selector:
```
.nhsuk-table-responsive {
... // the existing margin stuff
}
.nhsuk-table.reduced-table {
... // The new font stuff
}
```
Then we could also wonder about the new class being a bit more BEM'y something along the lines of `.nhsuk-table--reduced-font`

---

Yep good call